### DISCUSSION TITLE
3 small suggestions

### DISCUSSION THREAD

**Discussion by lucas-safy** [PaulWoitaschek/Voice#2543] [Ideas]
Hello and thank you very much for your application :)

I wanted to suggest 3 small changes which, if added, would improve the application in my opinion 

First of all, the "mark as read" button is too close to the bottom button on the phone. 
This makes it difficult to click. It just needs to be moved up a bit 

2nd: at the end of the playback of an audio file, it remains in the "In progress" section with the last 5 seconds.
I have to do a long click and then press hard on "mark commelux" to get them out of this section. 

3rd: there is a difference in naming between how the file appears in the "add file/folder to library" section and the library, particularly when displaying file titles. 

Best regards 
Lucas

**Comment by willemijns** [comment]
Plz create 1 idea per message
